prometheus-operator-standalone/values.yaml

```diff
@@ -41,11 +39,12 @@ prometheusOperator:
   ## This is mutually exclusive with denyNamespaces. Setting this to an empty object will disable the configuration
   ##
   namespaces: {}
-    # releaseNamespace: true
-    # additional:
-    # - kube-system
+  # releaseNamespace: true
+  # additional:
+  # - kube-system
```

Comment:
**Required**: We **MUST** undo this formatting change because if this is uncommented, the `prometheusOperator.namespaces.releaseNamespace` is going to be incorrectly indented and become `prometheusOperator.releaseNamespace` instead which is wrong.

See https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L1922.

Reply:
Thx! Fixed formatting issues.